src/Nitwork/Nitwork.h

```diff
@@ -12,8 +12,9 @@
     #include "MessageTypes.h"
 
     #define HEADER_SIZE sizeof(struct header_s)
-    #define TICKS_PER_SECOND 60
     #define ONE_SECOND 1000
+    #define TICKS_PER_SECOND 128
+    #define TICK (ONE_SECOND / (TICKS_PER_SECOND * ONE_SECOND))
```

Comment:
@coderabbitai its because one second is in milliseconds, and ticks per second is 128 so i need to convert it in miliseconds by doing * 1000 is it correct ?

Reply:
oh ok